android/app/src/main/res/layout/small_forecast_widget_layout.xml

```diff
@@ -92,17 +92,6 @@
             tools:text="Viranomaistiedote: 13.11.2024 klo: 11.00: Alueella Lauttasaaren ranni..."
             tools:visibility="visible" />
 
-        <TextView
-            android:id="@+id/updateTimeTextView"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center_horizontal"
-            android:textColor="@android:color/black"
-            android:textSize="10sp"
-            android:visibility="visible"
-            tools:text="11.34.24"
-            tools:ignore="SmallSp" />
-
```

Comment:
Can you put this back please?
I would like to see for testing purposes when the widget gets updated.
It can be set to GONE later.

Reply:
I think that we need to start testing before Christmas so next week we would need to remove these anyway. So that we can test proper design.

Anyway better way would be to log these and inspect times from log. There you would get longer timeline of updates and not just one time.

Reply:
Do we need a new implementation for the logging?

Reply:
Well, it's nothing new, because you have already done logging like this

`Log.d("Widget Location", "Permissions missing");`

Same way update times could be logged.

Reply:
I meant logging when Android Studio/Logcat is not connected, e.g. to a log file.

I still want to keep that update time TextView.
You can set its visibility to GONE.
I could still use it myself.

Reply:
Android is logging all the time and you can see old logs just connecting phone to PC/Mac. There also log viewers in Google Play store.

But I can keep it with visibility GONE if it helps you.

Reply:
Yeah, I do not want use log viewers from Google Play Store now.
In some other projects we had logs written to files, which was handy.